### PR TITLE
Feature/dt 1263 digital maryland mapping

### PR DESCRIPTION
**Pull request by mdellabitta**
Adds initial mapping for MD.

Currently setting this up in CQA. This is not ready to be merged.

### CODE REVIEW

File: profiles/maryland.pjs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "type": "oai_verbs",
+    "metadata_prefix": "oai_qdc",
+    "endpoint_url": "http://webconfig.digitalmaryland.org/oai/oai.php",
+    "contributor": {
+        "@id": "http://dp.la/api/contributor/digital-maryland",
+        "name": "Digital Maryland"
+    }, 
+    "enrichments_coll": [
+        "/set_context",
+        "/validate_mapv3"
```
Comment (moltude) [inline profiles/maryland.pjs]: Should be removed. Enrichment does not exist anymore

File: profiles/maryland.pjs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "maryland",
+    "type": "oai_verbs",
+    "metadata_prefix": "oai_qdc",
+    "endpoint_url": "http://webconfig.digitalmaryland.org/oai/oai.php",
+    "contributor": {
+        "@id": "http://dp.la/api/contributor/digital-maryland",
```
Comment (moltude) [inline profiles/maryland.pjs]: There is a discrepancy between the contributor and provider `@id` property value. 

`http://dp.la/api/contributor/maryland` vs `http://dp.la/api/contributor/digital-maryland` Not sure which one is correct.

File: lib/mappers/maryland_mapper.py [hunk 3]
```diff
@@ -11,7 +11,7 @@ def __init__(self, provider_data):
     def map_collection(self):
         collection = getprop(self.provider_data, "collection/title", True)
         if collection:
-            self.update_source_resource({"isPartOf": collection})
+            self.update_source_resource({"collection": {"title": collection}})
```
Comment (moltude) [inline lib/mappers/maryland_mapper.py]: should the collection dict include `id`, `@id`, and `description` properties even if they are empty?

Comment (mdellabitta) [inline lib/mappers/maryland_mapper.py]: My understanding of this from other work like this was the answer was no, or at least indeterminate. @markbreedlove?

File: profiles/maryland.pjs (hunk 1 above)
Comment (mdellabitta) [inline profiles/maryland.pjs]: 👍

**Comment by mdellabitta** [comment]
@molitude I made the changes you requested and rebased on develop.

**Comment by mdellabitta** [comment]
This needs to be squashed prior to merging.

**Comment by moltude** [comment]
Disregard PR.

**Pull request closed without merging**